common/rpc/interceptor/namespace.go

```diff
@@ -62,6 +63,10 @@ func GetNamespaceName(
 	req interface{},
 ) (namespace.Name, error) {
 	switch request := req.(type) {
+	case *workflowservice.RegisterNamespaceRequest:
```

Comment:
We would get a NamespaceNotFound error. We need the extra handling now so that we don't cache that the namespace was also not found when reading through

Reply:
Ah I see